cli/script/command-executor.ts

```diff
@@ -317,11 +357,44 @@ function deploymentHistory(command: cli.IDeploymentHistoryCommand): Promise<void
         })
         .then((deploymentId: string): Promise<Package[]> => {
             throwForInvalidDeploymentId(deploymentId, command.deploymentName, command.appName);
+            storedDeploymentId = deploymentId;
 
             return sdk.getPackageHistory(storedAppId, deploymentId);
         })
-        .then((packageHistory: Package[]): void => {
-            printDeploymentHistory(command, packageHistory);
+        .then((packageHistory: Package[]): Promise<void> => {
+            return sdk.getDeploymentMetrics(storedAppId, storedDeploymentId)
```

Comment:
(Open for discussion)

Would it be worth pre-fetching this on the server side to save the round trip? On the server side, you could just call the function that retrieves the metrics directly, and then move the code that merges the results below from the CLI into the server.